misc/json-to-record-converter/src/main/java/io/ballerina/jsonmapper/util/ListOperationUtils.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.jsonmapper.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Util methods for JSON to record direct converter.
+ *
+ * @since 2201.2.0
+ */
+public final class ListOperationUtils {
```

Comment:
WDYM by the name `ListOperationUtils`?

Reply:
Util methods which can be performed on List of elements